src/utils/tokenMetadata.ts

```diff
@@ -36,12 +36,13 @@ function createGetMultiAssetTxMintMetadataQuery(assets: PolicyIdAssetMapType) {
   const whereConditions = Object.keys(assets)
     .map((policyIdHex: string) => {
       const assetNames = assets?.[policyIdHex] ?? [];
+      const policID = `( encode(ma.policy, 'hex') = ($${index++})::varchar and`;
       const query = assetNames
-        .map((_) => 
-            `(encode(ma.name, 'hex') = ($${index++})::varchar
-          and encode(ma.policy, 'hex') = ($${index++})::varchar )`)
+        .map((_) => `encode(ma.name, 'hex') = ($${index++})::varchar`)
         .join(" or ");
-      return query;
+      const addPolicyId = `${policID} (${query}) )`;
```

Comment:
Right now we don't actually have any test that makes use of the OR behavior. Can we add a case for this in the test?

Reply:
Add tests for both